src/ml/neural_net/compute_context.hpp

```diff
@@ -16,6 +16,19 @@
 namespace turi {
 namespace neural_net {
 
+/** A struct to define all the parameters used to create the activity classifier
+ * model backend
+ */
+struct ac_parameters {
+  int batch_size;
+  int num_features;
+  int prediction_window;
+  int num_classes;
+  int num_predictions_per_chunk;
+  int random_seed;
+  float mode;
```

Comment:
As discussed offline, would be nice to make this a `bool` instead, maybe `is_training`?

Reply:
Okay